.github/workflows/golangci-lint.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.45.2
+          version: latest
```

Comment:
I have mixed feelings about not pinning a specific version. I fear the CI could start to break out of the blue as soon as a new version of golangci-lint is released.

However, feel free to go ahead. Do **not** consider this comment as blocking the merge